input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,265 @@ Example: Patient resource with interpreter required and language is known
 }
 ~~~
 
+### Representing body site, which may include laterality
+When using some AU Core profiles it may be desirable to represent a relevant body site, which may include laterality, associated with a record using CodeableConcept elements, notably:
+*  [AU Core Condition](StructureDefinition-au-core-condition.html) - with primary finding code `Condition.code` and `Condition.bodySite`
+*  [AU Core Procedure](StructureDefinition-au-core-procedure.html) - with primary procedure code `Procedure.code` and `Procedure.bodySite`
+
+When implementing representations of body site / laterality the following points can be noted and are considered in providing recommended representations for varied terminology usage in systems:
+* The `bodySite` element is not *Must Support* in AU Core profiles, there is no expectation to fill or meaningfully consume this element.
+* The `CodeableConcept.text` element is system populated and may reflect more specific detail than the `CodeableConcept.coding` concepts provided.
+
+In FHIR, body site and associated laterality can be recorded in various ways. Depending on the specific system approach to body site / laterality information recording this can lead to a number of scenarios needing to be implemented using AU Core profiles:
+
+~~~
+1. Primary finding/procedure code with body site and laterality as a pre-coordinated code.
+
+2. Primary finding/procedure code with body site (without laterality) as a pre-coordinated code, and a separate laterality coded qualifier.
+
+3. Coded body site with laterality and separate primary finding/procedure code.
+
+4. Coded body site without laterality and separate coded laterality qualifier and a primary finding/procedure code.
+~~~
+
+To support consistent representation the following is recommended for each of these cases, this approach can be applied to either Condition or Procedure profiles:
+
+1\. Primary `code` only (pre-coordinated body site including laterality)
+* For systems that have pre-coordinated coding describing a concept fully.
```

Comment:
Reformat this to be more similar to Missing Data advice so it becomes:

* For systems that have pre-coordinated coding describing a concept fully:
   * use only the `code` element to contain information on body site with laterality.